src/pool/srv_pool.c

```diff
@@ -1803,6 +1812,19 @@ ds_pool_connect_handler(crt_rpc_t *rpc)
 
 	ABT_rwlock_wrlock(svc->ps_lock);
 
+	/* Check if pool is being destroyed and not accepting connections */
+	d_iov_set(&value, &accept_connections, sizeof(accept_connections));
+	rc = rdb_tx_lookup(&tx, &svc->ps_root,
+			   &ds_pool_prop_accept_connections, &value);
+	if (rc != 0)
+		D_GOTO(out_lock, rc);
+	D_DEBUG(DF_DSMS, "pool "DF_UUID": accept_connections=%u\n",
+		DP_UUID(in->pci_op.pi_uuid), accept_connections);
+	if (accept_connections == 0) {
```

Comment:
[Optional] Since this is a Boolean-style (rather than scalar) variable, "if (accept_connections)" suffices and reads better.

Reply:
addressed in most recent patch

Reply:
Thanks.